scripts/scil_compute_seed_density_map.py

```diff
@@ -60,26 +54,22 @@ def main():
                      'must be greater than 0 and smaller or equal to {}'
                      .format(args.binary, max_))
 
-    # Load tractogram and load seeds
-    tracts_file = load(args.tractogram_filename, args.lazy_load)
-    if 'seeds' in tracts_file.tractogram.data_per_streamline:
-        seeds = tracts_file.tractogram.data_per_streamline['seeds']
+    # Load files and data. TRKs can have 'same' as reference
+    # Can handle streamlines outside of bbox
+    tractogram = load_tractogram(args.tractogram_filename, 'same',
+                                 bbox_valid_check=False)
+    # Streamlines are saved in RASMM but seeds are saved in VOX
+    # This might produce weird behavior with non-iso
+    tractogram.to_vox()
```

Comment:
Here you probably should add to_corner() and make sure that line 74 (`seed_voxel = np.round(seed).astype(np.int)`) fits.

Reply:
Fixed !

Reply:
@EmmaRenauld Why to corner ?

Reply:
I double checked with @frheault. It is ok !